src/components/MainHeaderHero/MainHeaderHero.tsx

```diff
@@ -107,7 +105,7 @@ export default function MainHeaderHero(props: Partial<MainHeaderHeroProps>) {
           media="(orientation: landscape) and (min-width: 1200px)"
           srcSet={srcSet.LANDSCAPE_MEDIUM}
         />
-        <img src={srcSet.LANDSCAPE_MEDIUM} className={styles.Image} alt={alt} />
+        <img src={srcSet.LANDSCAPE_MEDIUM} className={styles.Image} alt="" />
```

Comment:
Can you also remove the `alt` prop from the interface `MainHeaderHeroProps` ?

Reply:
Donde